### DISCUSSION TITLE
How best to help out?

### DISCUSSION THREAD

**Discussion by jcam** [zwave-js/zwave-js#1306] [General] [answered]
Ahoy!
This project looks awesome, and I'd love to help out. What are some areas that could use assistance, for someone who can code, and has dabbled about with zwave, but isn't an expert? Really hoping to see this project succeed where ozw did not.

Thanks!

**Comment by AlCalzone** [comment, marked as answer]
Hey,
IMO there are several ways you can help out. 

I think the most obvious way is implementing one of the **un-implemented CCs**. You can check [here](https://github.com/zwave-js/node-zwave-js/issues/6) which are implemented and in which version, [here](https://github.com/zwave-js/node-zwave-js/issues/729) which CCs are most requested, [here's](https://zwave-js.github.io/node-zwave-js/#/development/implementing-cc) a rough description how to do that and [here](https://github.com/zwave-js/node-zwave-js/tree/master/specs) you'll find the PDFs (SDS13781/SDS13782) outlining how the CCs are structured and how they should behave.
Stick to Application/Maintenance CCs, you **do not** want to get involved with the encapsulation CCs, and the Network stuff is not needed.

Then there are several open issues, some labeled "help wanted". For the others we should probably decide on a case-by-case basis if it makes sense.

Also (like you already did), assisting people on the issue tracker helps me focus on the actual work.
